unstructured/ingest/cli/cmds/astra.py

```diff
@@ -36,13 +36,6 @@ def get_cli_options() -> t.List[click.Option]:
                 "Note that the collection name must only include letters, "
                 "numbers, and underscores.",
             ),
-            click.Option(
-                ["--embedding-dimension"],
-                required=True,
-                default=384,
-                type=int,
-                help="The dimensionality of the embeddings",
-            ),
             click.Option(
                 ["--namespace"],
```

Comment:
"to write into" -> "to connect to" suggestion for --namespace as well

Reply:
Great idea. I'll give the help strings another look.